### PR TITLE
use CORS

### PR DESCRIPTION
**Pull request by KthProg**

### CODE REVIEW

**Comment by gamesguru** [comment]
Don't use python3 in the makefile

Instead copy it on your machine so both point to python3

**Comment by KthProg** [comment]
> Don't use python3 in the makefile
> 
> Instead copy it on your machine so both point to python3
> 
> venv should be easy to set up too and that will populate your PATH with an isolate python environment

Fixed it

**Comment by gamesguru** [comment]
You need to deploy dev to merge to master or prod.

git push origin HEAD:dev

File: ntserv/routes.py [hunk 1]
```diff
@@ -39,6 +40,9 @@
 app = sanic.Sanic(__module__)
 app.config.FORWARDED_SECRET = PROXY_SECRET
 
+app.config.CORS_ORIGINS = "http://localhost:3000,https://nutra.tk"
```
Comment (gamesguru) [inline ntserv/routes.py:43]: dev.nutra.tk

These should be imported from the init

**Review by gamesguru — CHANGES_REQUESTED** [review]

**Comment by gamesguru** [comment]
this is causing massive duplication on the auto-generated url map

these would have to be filtered out

is there a reason we need to support CORS today? Does fetch work without it

File: ntserv/routes.py [hunk 2]
```diff
@@ -78,6 +82,11 @@ async def _(request):
     return exc_req(get_nutrients, request)
 
 
+@app.route("/api/products")
+async def _(request):
+    return exc_req(get_nutrients, request)
```
Comment (gamesguru) [inline ntserv/routes.py:87]: is this needed?

**Pull request merged**